PKGBUILD

```diff
@@ -0,0 +1,50 @@
+# This is an example PKGBUILD file. Use this as a start to creating your own,
+# and remove these comments. For more information, see 'man PKGBUILD'.
+# NOTE: Please fill out the license field for your package! If it is unknown,
+# then please put 'unknown'.
+
+# Maintainer: Your Name <youremail@domain.com>
+pkgname="ros-melodic-libfranka"
+pkgver="0.7.1"
+_pkgver_patch=1
+pkgrel=1
+pkgdesc="C++ library for Franka Emika research robots "
+arch=(x86_64)
```

Comment:
Please use the complete `arch` list so that people can use it on other architectures.